### PR TITLE
1431 Upgrade canary storybook's to v7 in canary packages

### PR DESCRIPTION
**Pull request by zxh722**
## Summary of the changes
Both the Canary React and the Canary Web-Components Storybooks have now been updated to version 7.6.10. The storybook-static folder is now served on a http-server so that Loki can view and analyse the screenshots.

## Related issue
#1431 

## Checklist
- [x] Relevant unit tests and visual regression tests added. 
- [x] All acceptance criteria reviewed and met. 
- [x] A11y unit test added and yields no issues.
- [x] A11y plug-in on Storybook yields no issues. 
- [x] System light and dark mode tested with no loss of content. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [x] Browser support tested (Chrome, Safari, Firefox and Edge).

### CODE REVIEW

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Comment by ad9242** [comment]
if you click on a particular story, eg in canary-web-components > data-table > embedded & then click on the "Component API" tab, you just get a blank screen

it works ok if you are on the "Docs" node under data table

**Review by GCHQ-Developer-299 — APPROVED** [review]

File: packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx [hunk 1]
```diff
@@ -500,7 +499,7 @@ export class PaginationBar {
                   onInput={() => this.handleInputChange()}
                   max={this.totalPages}
                   min="1"
-                  validationInlineInternal
```
Comment (ad9242) [inline packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx:503]: i think the prop name is still `validationInlineInternal`

File: packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx [hunk 2]
```diff
@@ -486,7 +486,6 @@ export class PaginationBar {
                 label={this.inputError}
                 target={`#${PAGE_INPUT_FIELD_ID}`}
                 disableHover
-                disableClick
```
[the remark this answers is not in the document]
Comment (zxh722) [reply packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx]: Added back.

File: packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx (hunk 1 above)
Comment (zxh722) [reply packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx]: Added back.

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Review by ad9242 — APPROVED** [review]

**Pull request merged**